test/node/resendable-call-from-server.js

```diff
@@ -55,7 +61,7 @@ function connect(callback) {
       test.error(error, 'must connect without errors');
       connection = conn;
       client.session = session;
-      callback();
+      process.nextTick(callback);
```

Comment:
Why do we have it here? I'd say to put it it the callback itself 
`createServer(() => connect(() => process.nextTick(reconnect)));`